inode/config.go

```diff
@@ -94,6 +98,7 @@ type globalsStruct struct {
 	corruptionDetectedFalseBuf         []byte                        // holds serialized CorruptionDetected == false
 	versionV1Buf                       []byte                        // holds serialized Version            == V1
 	inodeRecDefaultPreambleBuf         []byte                        // holds concatenated corruptionDetectedFalseBuf & versionV1Buf
+	inodeSize                          uint64                        // size of in-memory inode struct
```

Comment:
As we've discussed... I do not agree with this number as it is actually only capturing the relatively small "constant" part of the size of an inode. I realize your and @charmster's differing opinions on this that "users think in terms of size, not count", but that is very, very hard to quantify. Sure, we could do it... but I doubt that's worth the effort to be honest. So I'd much prefer sticking with counts.

To that end, there is tremendous value in following a convention set by many other such mechanisms. All of those use counts... so it is bucking the pseudo standard as well to switch to size for just this one. It's kind of like somebody saying "hey, I like 6 character tab stops instead of the 4 character tab stops... so in the code I write I'll use 6". Does this make sense?